src/main/java/core/basesyntax/Engine.java

```diff
@@ -1,5 +1,25 @@
 package core.basesyntax;
 
-public class Engine {
+public class Engine implements Cloneable {
+
+    private String enginesType;
+    private String fuelGrade;
+    private int engineCapacity;
+    private int valvesPerCylinder;
+    private int power;
+
+    public Engine(String enginesType, String fuelGrade,
+                   int engineCapacity, int valvesPerCylinder, int power) {
+        this.enginesType = enginesType;
+        this.fuelGrade = fuelGrade;
+        this.engineCapacity = engineCapacity;
+        this.valvesPerCylinder = valvesPerCylinder;
+        this.power = power;
+    }
+
+    @Override
+    public Engine clone() throws CloneNotSupportedException {
```

Comment:
can we remove throws?

Reply:
yes, without it, too, passes, I did immediately without it, but then I looked at who they all accepted, so I added (

Reply:
we don't accept this task with throws)And it's bad to write off especially with no understanding